examples/ColumnGroupsExample.js

```diff
@@ -65,6 +65,13 @@ class ColumnGroupsExample extends React.Component {
             width={150}
           />
         </ColumnGroup>
+        <Column
```

Comment:
I'd don't think we should have examples of what we don't want people doing.  Instead could you remove this and put up the new column group header rendering.  I'd like to have that example and in place before merging this.  If you're afraid of the PR becoming to complicated feel free to change this PR to merge into a schrodinger:feat/widthhelper-refactoring branch which is based off master and then we can do the additional work off that before merging the whole thing into master.